source/jormungandr/jormungandr/street_network/asgard.py

```diff
@@ -99,6 +135,20 @@ def make_location(self, obj):
             place="", access_duration=0, lon=round(coord.lon, 5), lat=round(coord.lat, 5)
         )
 
+    def _get_language(self, language):
+        try:
+            return Languages[language]
+        except KeyError:
+            # self.log.error('Asgard parameters language={} not exist - force to english_us'.format(language))
+            return Languages.english_us
+
+    def get_language_parameter(self, request):
+        _language = request.get('_asgard_language', None)
+        if _language == None:
+            return Languages.english_us
+        else:
+            return self._get_language(_language.lower())
```

Comment:
```suggestion
        return request.get('_asgard_language', Languages.english_us).lower()
```
No need ```_get_language``` function , No?